frontend/src/modules/application/actions.js

```diff
@@ -24,4 +28,17 @@ export const createNewApplication = payload => async dispatch => {
             )
         }
     }
-}
\ No newline at end of file
+}
+
+// action used by applicant positions view
+export const fetchPositionsSuccess = payload => ({ type: FETCH_POSITIONS_SUCCESS, payload })
+export const fetchPositions = () => async dispatch => {
+    const res = await fetch("/api/v1/positions")
+    const data = await res.json()
+    if (res.status === 200) {
+        dispatch(fetchPositionsSuccess(data))
```

Comment:
Wouldn't it be better to await the JSON in here after you know the fetch was a success?